PurchasesTests/Networking/MockSKProductDiscount.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  MockSKProductDiscount.swift
+//  Purchases
+//
+//  Created by Joshua Liebowitz on 7/6/21.
+//  Copyright © 2021 Purchases. All rights reserved.
+//
+
+import Foundation
+import StoreKit
+
+@available(iOS 11.2, *)
+class MockSKProductDiscount: SKProductDiscount {
```

Comment:
this mock already exists https://github.com/RevenueCat/purchases-ios/blob/main/PurchasesTests/Mocks/MockProductDiscount.swift we should try to unify them

Reply:
Done ✅